sync/class.jetpack-sync-module-themes.php

```diff
@@ -359,6 +359,10 @@ function sync_sidebar_widgets_actions( $old_value, $new_value ) {
 				? $old_value[ $sidebar ]
 				: array();
 
+			if ( ! is_array( $new_widgets ) ) {
+				$new_widgets = array();
+			}
```

Comment:
@lezama @enejb asking for your feedback here because while this fixes the issue, maybe we don't need to through the rest of code if the `$new_widgets` is not an array, and then we could exit as early as [line 355](https://github.com/Automattic/jetpack/pull/7381/files#diff-37af365e9198e952a002bbf543cdcdcfR355).